index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse!</h1>
```

Comment:
- [ ] Kindly use the complete HTML boilerplate as it is how it is traditionally and conventionally used.

It is something like

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>HTML 5 Boilerplate</title>
    <link rel="stylesheet" href="style.css">
  </head>
  <body>
	
  </body>
</html>
```